frontend/src/views/supp-allowances/SupplementaryFormView.vue

```diff
@@ -75,32 +79,48 @@
             <IndigenousProgrammingAllowance
               v-if="panel.id === INDIGENOUS"
               :indigenousProgrammingModel="getModel(SUPPLEMENTARY_TYPES.INDIGENOUS, renewalTerm)"
-              @update="updateModel"
-              :formDisabled="currentTermDisabled || formDisabled" />
+              :formDisabled="currentTermDisabled || formDisabled"
+              @update="updateModel" />
             <SupportNeedsProgrammingAllowance
               v-if="panel.id === SUPPORT_NEEDS"
               :supportModel="getModel(SUPPLEMENTARY_TYPES.SUPPORT, renewalTerm)"
               :hasInclusionPolicy="currentOrg.hasInclusionPolicy"
               :formDisabled="currentTermDisabled || formDisabled"
               @update="updateModel" />
+          </v-expansion-panel-text>
+        </v-expansion-panel>
+
+        <v-expansion-panel-title class="tooltip py-6 mt-8" hide-actions>
+          <span class="header-label">Discretionary Services Allowance</span>
+        </v-expansion-panel-title>
+        <v-expansion-panel :key="DISCRETIONARY_PANEL.id" :value="DISCRETIONARY_PANEL.id">
+          <v-expansion-panel-title>
+            <span class="header-label">{{ DISCRETIONARY_PANEL.title }}</span>
+            <span v-if="isFundingActive(DISCRETIONARY_PANEL.id, renewalTerm)" class="active-label ml-9">Active</span>
+          </v-expansion-panel-title>
+          <v-expansion-panel-text>
             <TransportationAllowance
-              v-if="panel.id === TRANSPORTATION"
               :transportModels="getTransportModels(renewalTerm)"
               :formDisabled="currentTermDisabled || formDisabled"
               :renewalTerm="renewalTerm"
               :startDate="getStartDate(renewalTerm)"
               @update="updateModel"
-              @addModel="addBlankTransportModel"
-              @deleteModel="deleteTransportModel"
-              @deleteDocument="deleteDocument" />
+              @add-model="addBlankTransportModel"
+              @delete-model="deleteTransportModel"
+              @delete-document="deleteDocument" />
           </v-expansion-panel-text>
         </v-expansion-panel>
       </v-expansion-panels>
     </div>
+
     <div v-if="nextTermActive">
       <v-skeleton-loader v-if="loading" :loading="loading" type="table-tbody"></v-skeleton-loader>
       <v-expansion-panels v-else v-model="panel" multiple>
-        <v-expansion-panel v-for="panel in PANELS" :key="panel.id" :value="panel.id">
+        <v-expansion-panel-title class="tooltip py-6" hide-actions>
```

Comment:
See previous comment about usage of v-expansion-panel.

Reply:
done